src/components/Utils/Utils.css

```diff
@@ -0,0 +1,21 @@
+.favorite-not-selected:hover .heart-outline,
```

Comment:
We might want to change how `hover`s from css because it was a bit confusing when I hover and click the heart. I looked at Instagram "heart" and we can follow something similar. 
1. Unchecked heart + non-hover 
![image](https://user-images.githubusercontent.com/71732016/156067488-e3df7137-3e2c-4e43-a498-76f5995541c5.png)
2. Unchecked heart + hover 
![image](https://user-images.githubusercontent.com/71732016/156067524-a017c6f3-af55-4a94-95a3-8d20d3901683.png)
3. checked heart + non-hover / checked heart + hover 
![image](https://user-images.githubusercontent.com/71732016/156067568-75cef64f-3319-4840-b3c6-ee750c42cb4f.png)

let me know if you have any questions!

Reply:
Just made the change! Let me know if you think it's too subtle

Reply:
beautiful! thank you!